content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,271 @@
+emailservice            ClusterIP      172.20.136.4     <none>                                                                    5000/TCP                          28h
+frontend                ClusterIP      172.20.103.129   <none>                                                                    80/TCP                            40d
+frontend-external       LoadBalancer   172.20.16.223    a52d7c861c01c4466803a44373bc11dc-1387384363.us-east-2.elb.amazonaws.com   80:31482/TCP                      40d
+paymentservice          ClusterIP      172.20.244.25    <none>                                                                    50051/TCP                         28h
+productcatalogservice   ClusterIP      172.20.147.163   <none>                                                                    3550/TCP                          40d
+recommendationservice   ClusterIP      172.20.83.157    <none>                                                                    8080/TCP                          40d
+redis-cart              ClusterIP      172.20.179.232   <none>                                                                    6379/TCP                          40d
+shippingservice         ClusterIP      172.20.16.129    <none>                                                                    50051/TCP                         28h
+skupper                 LoadBalancer   172.20.111.44    aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com    8080:31907/TCP,8081:30027/TCP     40d
+skupper-router          LoadBalancer   172.20.182.70    acaedc6978d3b453b8555d6dead90943-1598691456.us-east-2.elb.amazonaws.com   55671:30272/TCP,45671:32499/TCP   40d
+skupper-router-local    ClusterIP      172.20.175.145   <none>                                                                    5671/TCP                          40d
+skupper-router-local    ClusterIP      172.20.249.51    <none>                                                                    5671/TCP                          35m                       
+```
+
+T﻿he *Online Boutique* application can be accessed from the assigned LoadBalancing host name **a52d7c861c01c4466803a44373bc11dc-1387384363.us-east-2.elb.amazonaws.com**:
+
+![](/img/online-boutique-frontend.png)
+
+You can start shopping by adding items to the shopping cart, creating your shipping address and choosing the payment method. Please note that both the payment and the shipping services are running from the private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise.
+
+![](/img/online-boutique-payment.png)
+
+You can then place order to complete your shopping.
+
+![](/img/online-boutique-order.png)
+
+## Next Steps
+
+This blog post described the process of deploying the *Online Boutique* application as a hybrid app across both public EKS cluster in AWS and private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise environment. 
```

Comment:
Add the word "a" in two places in this sentence:

"... as a hybrid app across both **a** public EKS cluster in AWS and **a** private Kubernetes..."